schema.sql

```diff
@@ -63,4 +63,13 @@ CREATE TABLE visits (
     date_of_visit date,
     CONSTRAINT fk_animals FOREIGN KEY(animals_id) REFERENCES animals(id),
     CONSTRAINT fk_vets FOREIGN KEY(vets_id) REFERENCES vets(id)
-);
\ No newline at end of file
+);
+
+-- Add an email column to your owners table
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+CREATE INDEX animals_id  ON visits(animals_id);
+CREATE INDEX vets_id  ON visits(vets_id);
```

Comment:
`For SELECT * FROM visits where vets_id = 2;`:
The second screenshot which is named `After` should have an execution time lower than 1000ms. Please add the screenshot which increases performance after adding the index to the foreign key.